etc/lmod-setup.sh

```diff
@@ -43,11 +43,8 @@ elif [ "$L_MACHINE" = singularity ]; then
    module purge
 
 elif [ "$L_MACHINE" = gaea ]; then
-   export BASH_ENV="/lustre/f2/dev/role.epic/contrib/apps/lmod/lmod/init/bash"
-   source $BASH_ENV
+   source /lustre/f2/dev/role.epic/contrib/Lmod_init.sh
 
-   export LMOD_SYSTEM_DEFAULT_MODULES="modules/3.2.11.4"
-   module --initial_load --no_redirect restore
```

Comment:
Are the cray modules no longer required?

Reply:
The list of modules is loaded by the initialization script Lmod_init.sh, depending on modules loaded by default by the system.

Reply:
Got it.